examples/centos.Dockerfile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos:7
+FROM centos:7 AS base
```

Comment:
Ah I guess it was related to ci failures.

My case in https://github.com/docker/buildkit-syft-scanner/issues/20 was generating the sbom but was empty.

Maybe we could check some packages are effectively there in our e2e tests?

Reply:
Yeah, this is kinda related, but wasn't the thing causing the failure :eyes: The SBOM was still empty, but it did have the wrong name of `stage-0` instead of `base` like the others.

Reply:
Ah yes right!